docs/contributing/stacks.md

```diff
@@ -89,11 +89,11 @@ The cookiecutter template comes with a `.github/workflows/docker.yml` file, whic
 
 2. Commit your changes and push to GitHub.
 3. Head back to your repository and click on the **Actions** tab.
-   ![GitHub actions tab screenshot](../_static/github-actions-tab.png)
+   ![GitHub page for jupyter/docker-stacks with the Actions tab active and a rectangle around the "Build Docker Images" workflow in the UI](../_static/github-actions-tab.png)
```

Comment:
I don't think this is visible in the docs, but I'm ok with longer descriptions to make them more clear in the source code.

Reply:
It is not visible but it should be generated as alt-text for those using screen readers or assistive tech. So definitely worth having better descriptions

Reply:
Oh, I see, thanks for the explanation.